### DISCUSSION TITLE
Add Resource Limits to Docker-Compose deployments.

### DISCUSSION THREAD

**Discussion by iamEvanYT** [coollabsio/coolify#3977] [Improvement Requests]
### Description

I think it used to be able to set resource limits through coolify for docker-compose deployments, but it's gone.

### Minimal Reproduction (if possible, example repository)

![Screenshot 2024-03-28 at 1 28 09 PM](https://github.com/coollabsio/coolify/assets/47493765/84e24f74-db7f-42de-98f6-dd4383f31f1d)

### Version

v4.0.0-beta.248

**Comment by MrSnoozles** [comment]
Would be useful to be able to set and see resource limits for docker-compose deployments directly in the GUI
